WrathCombo/Combos/PvP/DRKPVP.cs

```diff
@@ -60,7 +60,7 @@ protected override uint Invoke(uint actionID, uint lastComboActionID, float comb
 
                         if (canWeave)
                         {
-                            if (IsEnabled(CustomComboPreset.DRKPvP_BlackestNight) && ActionReady(BlackestNight) && !HasEffect(Buffs.BlackestNight))
+                            if (IsEnabled(CustomComboPreset.DRKPvP_BlackestNight) && ActionReady(BlackestNight) && !HasEffect(Buffs.BlackestNight) && !WasLastAbility(BlackestNight))
```

Comment:
I'd suggest replacing `WasLastAbility` with `WasLastAction` or it won't use consecutive Blackest Nights between weaponskills.

Reply:
TBN is an ability. You dont want to use it twice in a single gcd. it is overwriting itself. it literally fixing that because some server ticks are bad